examples/weekly-summary-email/steps/run/.env

```diff
@@ -0,0 +1 @@
+SENDGRID_API_KEY=""
```

Comment:
This doesn't work for me here - the `.env` files only seem to be consumed if they're placed in the function root (`examples/weekly-summary-email`) as opposed to the step root.

Not sure if this is a bug or just not something that we thought of the first time around?

It'd be great to have it look at both locations and merge the env so we get:

- "Global" function `.env`
- Step-specific `.env` (which overwrites any function env vars with the same names)

To fix this now we can just move it up to the function root.